src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -109,11 +108,8 @@ static void LogFindResults(ActiveLicenseFindResult result)
 
                 Logger.Debug(report.ToString());
             }
-            else
-            {
-                var report = string.Join(Environment.NewLine, result.SelectedLicenseReport);
-                Logger.Info(report);
-            }
+
+            Logger.Info(string.Join(Environment.NewLine, result.SelectedLicenseReport));
```

Comment:
@bording looking at what the license source packages does I guess there will be some duplicate info btw the DEBUG and INFO logs but I think we can live with that?

Reply:
I'm a bit confused as to what change this PR is intending to make. 

Why have you removed the condition for this entry?

Reply:
Discussed with @bording and we're all good with proceeding with this

Reply:
Got on a call, all makes sense now, and yes the duplication of log entries seems fine in this case.